sidebars.js

```diff
@@ -68,6 +66,27 @@ const sidebars = {
   ],
   developersSidebar: [
       {
+        type: "category",
+        collapsed: false,
+        label: "Build on Linea",
```

Comment:
Is it necessary to have this top-level category (`Build on Linea`) in the sidebar in order for the rest of the sidebar to work? Because it doesn't really seem to have any function

Reply:
Fixed in [this commit](https://github.com/Consensys/doc.zk-evm/pull/465/commits/17ce3c1d2efb3b1133d8a9e6185ae1aba9784007)